tox.ini

```
@@ -22,3 +22,13 @@ paths =
 # NB: (Pinned) pylint is installed as part of the lektor[test] extras
 commands =
     pylint {posargs:lektor setup.py tests}
+
+[flake8]
+max-line-length = 100
+ignore =
```

1) use
```suggestion
extend-ignore =
```
to append to the default list of ignores instead of replacing it.
2) it's usually _much_ easier to find this config if it's in a dedicated file

---

> 1. use extend-ignore

+1

Thank you




> 2) it's usually _much_ easier to find this config if it's in a dedicated file

You mean put the flake8 config in `.flake8`, I assume.
Personally, I don't like project-local configs in dot files, as I tend not to notice that they are there.   But I don't have strong feelings.  I do kind of like to have all the dev/test tool config in one place — but, even if you agree with me on that, that's not always possible these days.
I don't have strong feelings either way.  Tell me where to put the flake8 config and I'll put it there.

---

> Personally, I don't like project-local configs in dot files

:+1:

---

Leaving flake8 config in `tox.ini` for now.

I've switched to using `extend-ignore`.